javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const width = 5;
+const height = 10;
+const area = width * height;
+console.log('the value of area: ', area);
+console.log('typeof area: ', typeof area);
+const bill = 50;
+const payment = 100;
+const change = payment - bill;
+console.log('the value of change: ', change);
+console.log('typeof change: ', typeof change);
+const quizzes = 100;
+const midterm = 99;
+const final = 100;
+const grade = (quizzes + midterm + final) / 3;
+console.log('the value of grade: ', grade);
+console.log('the typeof grade: ', typeof grade);
+const firstName = 'Walter';
+const lastName = 'White';
+const fullName = firstName + ' ' + lastName;
+console.log('the value of fullName: ', fullName);
+console.log('the typeof fullName: ', typeof fullName);
+const pH = 8;
+const isAcidic = pH < 7;
+console.log('the value of isAcidic: ', isAcidic);
+console.log('the typeof of isAcidic: ', typeof isAcidic);
+const headCount = 9000;
+const isSparta = headCount === 300;
+console.log('value of isSparta: ', isSparta);
+console.log('typeof isSparta: ', typeof isSparta);
+const motto = fullName + ' is the GOAT';
```

Comment:
You should be using the `+=` or Addition assignment operator for this one!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition_assignment